graylog2-web-interface/webpack/vendor-module-ids.json

```diff
@@ -1078,7 +1078,7 @@
       "node_modules/core-js/library/modules/_parse-int.js": 147,
       "node_modules/core-js/library/modules/_string-trim.js": 148,
       "node_modules/react-transition-group/utils/PropTypes.js": 251,
-      "node_modules/@babel/runtime/helpers/extends.js": 441,
+      "node_modules/@babel/runtime/helpers/extends.js": 440,
```

Comment:
This change should not be in here, afaics. There are no changes to packages included in the vendor bundle in this PR, so it should not change the vendor module ids.

Reply:
Agreed, will take care of it 👍